### DISCUSSION TITLE
Nondimensionalisation in contact mechanics

### DISCUSSION THREAD

**Discussion by sannant** [ContactEngineering/topobank#888] [Q&A]
### FAQ: 

The website returns the fractional contact area (A/A0) as a function of normalized pressure (p/E*).

If I want to calculate the contact mechanics for only one surface contacting against an elastic flat, what would the value of E* be (i.e. what are the values of Elastic Modulus and Poisson's ratio for the elastic half space)?

The simulations return the nondimensional values p / E^* . 

### Answer 

So this means A/A0 vs. p / E^* curve informs you about the pressure area relationship for any elastic substrate.

I.e. for steel with E* around 200MPa, you multiply the nondimensional mean pressure values returned by the webapp by 200 MPa to get the physical mean pressures for a rough steel contact.

**Comment by EthanDeng07** [comment]
good one!
